guides/common/modules/proc_using-ansible-job-templates-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions{context}"]
```

Comment:
```suggestion
[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions"]
```

or

```suggestion
[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions_{context}"]
```

AFAIK it would be valid to _no_ use the context attribute in the anchor if it's not necessary aka. if you don't intend to include this procudure twice in the same guide. Depends on whether this is part of "Managing Hosts" twice: once for REX, once for Ansible?

Reply:
Looks like it's just for Ansible. I will remove the context attribute.